src/request/index.ts

```diff
@@ -247,14 +247,23 @@ export function jsonRequest(
 		[soapRequestName]: soapCommandBody(options)
 	};
 
+	let fetchOptions;
+
+	if (requestOptions.fetchOptions) {
+		fetchOptions = {
+			signal: requestOptions.fetchOptions.signal
```

Comment:
what is signal?

Reply:
can't we just send it as requestOptions.signal? instead of adding fetchOptions layer?

Reply:
its for abortcontroller, so ignore above comments